src/sentry/eventstore/models.py

```diff
@@ -772,6 +774,8 @@ def __getitem__(self, name: str) -> str:
             return self.event.title
         elif name == "issueType":
             return cast(str, GROUP_TYPE_TO_TEXT.get(self.event.group.issue_type, "Issue"))
+        elif name == "issueTitle":
+            return cast(str, self.event.occurrence.issue_title)
```

Comment:
Looks good for now. I wonder if down the road we will just want to change `"title"` to be
`name = self.event.occurrence.issue_title if self.occurrence else self.event.title`

Reply:
I like this better actually, because it can just fall back to the default template and use the title. Updated :)